src/applications/ivc-champva/10-7959a/chapters/claimInformation.js

```diff
@@ -0,0 +1,325 @@
+      title:
+        'Is this a claim for an injury or condition caused by an auto-related accident?',
+      updateUiSchema: formData => {
+        return {
+          'ui:options': {
+            hint: `Depending on your answer, we may contact ${nameWording(
+              formData,
+              true,
+              false,
+              true,
+            )} auto insurance provider to determine if they paid any amount for this care.`,
+          },
+        };
+      },
+    }),
+  },
+  schema: {
+    type: 'object',
+    required: ['claimIsAutoRelated'],
+    properties: {
+      titleSchema,
+      claimIsAutoRelated: yesNoSchema,
+    },
+  },
+};
+
+export const medicalClaimUploadSchema = {
+  CustomPage: props => FileFieldCustom({ ...props, requiredFiles: [] }),
+  CustomPageReview: null,
+  uiSchema: {
+    ...titleUI('Upload supporting documents', ({ formData }) => (
+      <>
+        You’ll need to submit a copy of an itemized billing statement for this
+        claim.
+        <br />
+        <p>
+          <b>Documentation must include all of this information:</b>
+        </p>
+        <ul>
+          <li>
+            {nameWording(formData, true, true, true)} full name, date of birth,
+            and Social Security Number.
+          </li>
+          <li>
+            {nameWording(formData, true, true, true)} provider’s full name,
+            medical title, office address, billing address, and tax
+            identification number.
+          </li>
+          <li>
+            A list of diagnosis and procedure codes for the care. This includes
+            DX, CPT, or HCPS codes.
+          </li>
+          <li>
+            A list of charges for {nameWording(formData, true, false, true)}{' '}
+            care, and the dates when {nameWording(formData, false, false, true)}{' '}
+            got the care.
+          </li>
+        </ul>
+        <p>
+          <b>Note:</b>
+          &nbsp; You may need to ask your provider for a statement that has all
+          of the information listed here. The statement must include all of this
+          information in order for us to process this claim.
+        </p>
+      </>
+    )),
+    ...fileUploadBlurbCustom(
+      <li>
+        You can also upload any other documentation you believe is related to
+        your medical claim
+      </li>,
+      additionalNotesClaims,
+    ),
+    medicalUpload: fileUploadUI({
+      label: 'Upload supporting document',
+      attachmentName: true,
+    }),
+  },
+  schema: {
+    type: 'object',
+    required: ['medicalUpload'],
+    properties: {
+      titleSchema,
+      'view:fileUploadBlurb': blankSchema,
+      medicalUpload: {
+        type: 'array',
+        minItems: 1,
+        items: {
+          type: 'object',
+          properties: {
+            name: {
+              type: 'string',
+            },
+          },
+        },
+      },
+    },
+  },
+};
+
+export const eobUploadSchema = isPrimary => {
```

Comment:
This page schema is used for the two EOB upload screens - by passing `true` or `false` we control which health insurance company name is shown in the header.